hazelcast/src/main/java/com/hazelcast/cardinality/impl/CardinalityEstimatorService.java

```diff
@@ -177,4 +210,84 @@ public String getQuorumName(String name) {
                 quorumConfigConstructor);
         return quorumName == NULL_OBJECT ? null : (String) quorumName;
     }
+
+    private SplitBrainMergePolicy getMergePolicy(String name) {
+        String mergePolicyName = nodeEngine.getConfig().findCardinalityEstimatorConfig(name).getMergePolicyConfig().getPolicy();
```

Comment:
Can this throw a NPE?

Reply:
Not really, not here actually. It could later when trying to find the policy in the next line, but thats wanted behavior (if config is wrong).